numpy/ma/extras.py

```diff
@@ -736,7 +736,7 @@ def _median(a, axis=None, out=None, overwrite_input=False):
     # duplicate high if odd number of elements so mean does nothing
     odd = counts % 2 == 1
     if asorted.ndim > 1:
-        np.copyto(low, high, where=odd)
+        low[odd] = high[odd]
```

Comment:
There were problems when odd was a scalar or scalar array. Perhaps that no longer happens with the other changes...